libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.h

```diff
@@ -176,11 +176,6 @@ extern NSString * const kSFUserLoggedInNotification;
 extern NSString * const kSFAuthenticationManagerFinishedNotification;
 
 /**
- Key for account object present in user info in notifications.
- */
-extern NSString * const kSFUserAccountKey;
```

Comment:
I reverted the fix submitted by the external contributor (that line came with that fix)